fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -5360,6 +5360,16 @@ public ParseNode visitSimpleFunctionCall(StarRocksParser.SimpleFunctionCallConte
             return new MapExpr(Type.ANY_MAP, exprs, pos);
         }
 
+        if (functionName.equals(FunctionSet.LPAD) || functionName.equals(FunctionSet.RPAD)) {
```

Comment:
excellent!! in future, we can handle the default parameters of other functions in this way, moreover, we can provide enum parameters for some functions intuitively.